transform/operations_test.go

```diff
@@ -375,21 +400,74 @@ func TestTimeParse(t *testing.T) {
 	runOpTests(t, func() transformOperation { return &timeParse{} }, tests)
 }
 
+func TestCurrentTime(t *testing.T) {
+	tests := []opTests{
+		{
+			description: "Simple working case",
+			args:        map[string]string{"format": time.RFC3339},
+			want:        time.Now().Format(time.RFC3339),
+		},
+		{
+			description: "Missing arg",
+			args:        map[string]string{},
+			wantInitErr: true,
+		},
+		{
+			description: "Too many args",
+			args:        map[string]string{"format": time.RFC3339, "cookies": "failure"},
+			wantInitErr: true,
+		},
+	}
+
+	for _, test := range tests {
```

Comment:
Use `t.Run(test.description, ...` inside the loop and replace your `t.Error()`s in the loop with `t.Fatal()`